### PR TITLE
adding unit at axis label

### PR DESCRIPTION
**Pull request by moontrip**
made a function to return axis label with unit if exists: for now, only variable with number type will show unit(s) in axes. As an example, only scatter plot viz is used to test: if acceptable then I will apply this to other vizs. A screenshot is attached for reference. Also, variable selector for the facet variable is removed as shown in the screenshot.

axislabelunit1
.

### CODE REVIEW

File: src/lib/core/utils/axisLabelUnit.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Variable } from '../types/study';
```
Comment (jtlong3rd) [inline src/lib/core/utils/axisLabelUnit.ts:1]: Nit: name this file `axis-label-unit`, to be consistent with the names of the other files in `utils`.

**Review by jtlong3rd — COMMENTED** [review]
Aside from one small inline comment, I think this is good to apply to the other viz"es"!

File: src/lib/core/utils/axisLabelUnit.ts (hunk 1 above)
Comment (moontrip) [reply src/lib/core/utils/axisLabelUnit.ts]: Thank for you review 👍 I have changed the file name

**Review by dmfalke — COMMENTED** [review]
This looks good to me. Feel free to proceed with other visualizations!

File: src/lib/core/utils/axis-label-unit.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Variable } from '../types/study';
+
+export function axisLabelUnit(
```
Comment (bobular) [inline src/lib/core/utils/axis-label-unit.ts:3]: Would you mind renaming this `axisLabelWithUnit` ?

Comment (moontrip) [reply src/lib/core/utils/axis-label-unit.ts:3]: @bobular No problem! I can change the function name :)

Comment (moontrip) [reply src/lib/core/utils/axis-label-unit.ts]: @bobular I have changed the name as suggested

**Comment by moontrip** [comment]
@jtlong3rd and @dmfalke I have updated this PR by addressing a couple of things as follows:

a) changed function name following Bob's suggestion
b) simplified the use of `findEntityAndVariable` function for all vizs by referring to what Bob did for boxplot viz. Tested several cases to ensure that they are working fine with changes
c) added axis label unit to all vizs
d) facet variable selector is removed at all vizs

**Review by jtlong3rd — APPROVED** [review]

**Review by dmfalke — APPROVED** [review]

**Pull request merged**